frontend/app/views/fragments/global/controlNavigation.scala.html

```diff
@@ -75,16 +74,14 @@
             </nav>
         </li>
     }
-    @if(mainSiteLink) {
-        <li class="nav-control__item nav-control__item--last hidden-mobile">
-            <a class="control" href="http://theguardian.com/" role="menuitem">
-                <span class="control__item">
-                    @fragments.inlineIcon("g-mark", List("icon-inline--large"))
-                </span>
-                <span class="control__item">
-                    <span class="control__info">the guardian</span>
-                </span>
-            </a>
-        </li>
-    }
+    <li class="nav-control__item nav-control__item--last hidden-mobile">
```

Comment:
Ah, no, sorry, I should have made that clearer; this code isn't specific to the simple header. It was always shown in the main header, I just [added a flag](https://github.com/guardian/membership-frontend/pull/1368/files#diff-a50ebb05028e3a9ba0785d1ba9dbc24b) so we could have the option of hiding it for the simple one.

However, now that the simple header no longer uses anything from that file, the flag is no longer necessary 😄.